src/services/namex.services.ts

```diff
@@ -46,8 +46,16 @@ export default class NamexServices {
     data: NameRequestI
   ): Promise<NameRequestI> {
     try {
-      // const requestAction = getters.getRequestActionCd
-      const requestAction = RequestActions.find(request => request.value === requestActionCd)
+      // create local Request Actions list with special case Resubmit
```

Comment:
It would have been really messy to change the global Request Actions list, since it is used in several places, and notably to populate the menus; the Resubmit action could not be displayed there.